frontend/common/src/helpers/skillUtils.ts

```diff
@@ -0,0 +1,39 @@
+import { flatMap, uniqBy } from "lodash";
+import { Skill, SkillFamily } from "../api/generated";
```

Comment:
If you like the other suggestion, you'll need this import.

```suggestion
import { Skill, SkillFamily } from "../api/generated";
import { notEmpty } from "./util";
```